### PR TITLE
ci: do not run quickstarts with old libraries

### PR DESCRIPTION
**Pull request by coryan**
We will compile, but not run, quickstarts compiled against previous
versions of the library.

Part of the work for #6221

---
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/8345)

### CODE REVIEW

File: ci/kokoro/windows/build-quickstart-bazel.ps1 [hunk 1]
```diff
@@ -112,65 +112,6 @@ if (-not (Test-Path env:KOKORO_GFILE_DIR)) {
...
-        bazelisk $common_flags fetch $build_flags ... $external
-        if ($LastExitCode -eq 0) {
-            break
-        }
-        Start-Sleep -Seconds (60 * $_)
-    }
-
-    Write-Host -ForegroundColor Yellow "`n$(Get-Date -Format o) " `
-        "Compiling quickstart for ${library}"
-    bazelisk $common_flags build $build_flags $target
```
[the remark this answers is not in the document]
Comment (coryan) [reply ci/kokoro/windows/build-quickstart-bazel.ps1:151]: Yes, we want to build the quickstarts with the old libraries, we just do not want to run them.

Maybe you want to argue that we do not need to build them against old libraries at all?  That could be a separate cleanup, but I think we should: there is special code for each platform in our per-quickstart `WORKSPACE`, `BUILD`, and/or `CMakeLists.txt` files

[the remark this answers is not in the document]
Comment (dbolduc) [reply ci/kokoro/windows/build-quickstart-bazel.ps1:151]: I take it they are built here.
https://github.com/googleapis/google-cloud-cpp/blob/910f302dea13e066e20597e9b9821eb9bb193918/ci/kokoro/windows/build-bazel.ps1#L130

**Review by dbolduc — APPROVED** [review]
I don't know what's happening, but I won't let that stop you. Maybe you want to ask someone else to look at this

File: ci/kokoro/windows/build-quickstart-bazel.ps1 (hunk 1 above)
[the remark this answers is not in the document]
Comment (coryan) [reply ci/kokoro/windows/build-quickstart-bazel.ps1:151]: That build is against the current version of the libraries, using the top-level WORKSPACE file.  This build (`build-quickstart-bazel`) builds against the latest release, using the WORKSPACE file in the `google/cloud/${library}/quickstart` directory.

[the remark this answers is not in the document]
Comment (dbolduc) [reply ci/kokoro/windows/build-quickstart-bazel.ps1:151]: I am pretty sure we deleted the line that builds the code in this PR and our windows `quickstart-bazel` has not been doing anything for the last 7 months.

https://github.com/googleapis/google-cloud-cpp/blob/main/ci/kokoro/windows/build-quickstart-bazel.ps1

I only see `bazelisk version` and `bazelisk shutdown`. Surely we should be `build`ing `run`ning or `test`ing

File: ci/kokoro/windows/build-quickstart-cmake.ps1 [hunk 2]
```diff
@@ -64,53 +64,4 @@ if (-not (Test-Path env:KOKORO_GFILE_DIR)) {
...
-    # Configure CMake and create the build directory.
-    Write-Host -ForegroundColor Yellow "`n$(Get-Date -Format o) Configuring CMake with $cmake_flags"
-    cmake $cmake_flags
-    if ($LastExitCode) {
-        Write-Host -ForegroundColor Red "cmake config failed with exit code $LastExitCode"
-        Exit ${LastExitCode}
-    }
-
-    Write-Host -ForegroundColor Yellow "`n$(Get-Date -Format o) Compiling with CMake $env:CONFIG"
-    cmake --build "${binary_dir}" --config $env:CONFIG
```
[the remark this answers is not in the document]
Comment (dbolduc) [reply ci/kokoro/windows/build-quickstart-cmake.ps1:97]: Same here. This build doesn't do anything beyond running the `build-cmake-dependencies.ps1` script. Should we just delete them?

**Pull request merged**